client/src/components/itemPriceDetails.jsx

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
 import { PriceText, BlueText, PriceSavedText, WithDealPriceText, ListPriceText } from '../style.js';
+import moment from 'moment';
 
 
-const ItemPriceDetails = (props) => (
-  <div>
-    <PriceText>List Price: </PriceText> <ListPriceText>${Math.floor(props.price + 10).toFixed(2)}</ListPriceText>
-    <br></br>
-    <PriceText>With Deal:</PriceText> <WithDealPriceText>${props.price}</WithDealPriceText>  Prime Free Delivery
-    <br></br>
-    <PriceText>You Save:</PriceText> <PriceSavedText>${10.00.toFixed(2)} ({Math.floor((((props.price + 10) - (props.price)) / (props.price + 10)) * 100)}%) </PriceSavedText>
-    <div>Order it now.</div>
-    <br></br>
-    <br></br>
-    <div>FREE Delivery <b>Jan 29 - Feb 3</b> for Prime members</div>
-    <BlueText>Details</BlueText>
-    <br></br>
-    <div>Ships from and sold by {props.brand}.com Services LLC.</div>
-  </div>
-);
+const ItemPriceDetails = (props) => {
+  let available = true;
+  if (props.inventory === 0) {
+    available = false;
+  }
+
+  const currentDate = moment().format('MMM Do');
+  const futureDate = moment().add(5, 'days').format('MMM Do');
+
+  return (
+    <div>
+      <PriceText>List Price: </PriceText> <ListPriceText>${Math.floor(props.price + 10).toFixed(2)}</ListPriceText>
+      <br></br>
+      <PriceText>With Deal:</PriceText> <WithDealPriceText>${props.price}</WithDealPriceText>  Prime Free Delivery
+      <br></br>
+      <PriceText>You Save:</PriceText> <PriceSavedText>${10.00.toFixed(2)} ({Math.floor((((props.price + 10) - (props.price)) / (props.price + 10)) * 100)}%) </PriceSavedText>
```

Comment:
Why are you adding 10 in props.price?

Reply:
This was a quick fix, that if I have time I will try and work on to come up with a better way of displaying the amount saved. Right now 10 is just a "magic number" I picked. Basically every item has a sale price and I decided to have every item save 10 dollars.

Reply:
got it